src/lib/svg_text_utils.js

```diff
@@ -400,7 +400,12 @@ function buildSVGText(containerNode, str) {
     }
 
     function exitNode(type) {
+        // A bare closing tag can't close the root node. If we encounter this it
+        // means there's an extra closing tag that can just be ignored:
+        if(nodeStack.length === 1) return;
```

Comment:
That seems reasonable. Do those log messages show up on-screen? Does that seem desirable?